packages/fast-colors/src/rgb.spec.ts

```
@@ -0,0 +1,186 @@
+    });
+});
+
+describe("Valid 8 digit hexits", (): void => {
+    test("Should return rgba(255, 255, 255, 1)", (): void => {
+        expect(hexToRgb("#FFFFFFFF")).toBe("rgba(255, 255, 255, 1)");
+    });
+    test("Should return rgba(0, 0, 0, 0)", (): void => {
+        expect(hexToRgb("#00000000")).toBe("rgba(0, 0, 0, 0)");
+    });
+    test("Should return rgba(255, 0, 0, 0.8)", (): void => {
+        expect(hexToRgb("#FF0000CC")).toBe("rgba(255, 0, 0, 0.8)");
+    });
+    test("Should return rgba(253, 224, 20, 0.5764705882352941)", (): void => {
+        expect(hexToRgb("#Fde01493")).toBe("rgba(253, 224, 20, 0.5764705882352941)");
+    });
+    test("Should return rgba(25, 115, 40, 0.4392156862745098)", (): void => {
+        expect(hexToRgb("#19732870")).toBe("rgba(25, 115, 40, 0.4392156862745098)");
+    });
+    test("Should return rgba(171, 254, 202, 0.8588235294117647)", (): void => {
+        expect(hexToRgb("#ABFECADB")).toBe("rgba(171, 254, 202, 0.8588235294117647)");
+    });
+    test("Should return rgba(155, 206, 135, 0.6862745098039216)", (): void => {
+        expect(hexToRgb("#9Bce87Af")).toBe("rgba(155, 206, 135, 0.6862745098039216)");
+    });
+    test("Should return rgba(169, 131, 71, 0.807843137254902)", (): void => {
+        expect(hexToRgb("#a98347ce")).toBe("rgba(169, 131, 71, 0.807843137254902)");
+    });
+    test("Should return rgba(239, 202, 137, 0.17254901960784313)", (): void => {
+        expect(hexToRgb("#efca892c")).toBe("rgba(239, 202, 137, 0.17254901960784313)");
+    });
+    test("Should return rgba(117, 131, 201, 0.2196078431372549", (): void => {
+        expect(hexToRgb("#7583c938")).toBe("rgba(117, 131, 201, 0.2196078431372549)");
+    });
+});
+
+describe("Invalid hexits", (): void => {
+    test("Should throw an error", (): void => {
+        expect((): void => {
+            hexToRgb("");
+        }).toThrow();
+    });
+    test("Should throw an error", (): void => {
```

I think these could be more specific and collected into groupings eg. "should throw an error if a non-alphanumeric value is used", "should throw if the alphanumeric values provided are out of range", you can add multiple expect to each test where it makes sense.